include/avif/internal.h

```diff
@@ -647,6 +647,10 @@ typedef size_t avifBoxMarker;
 
 typedef struct avifBoxHeader
 {
+    // If set to AVIF_TRUE, then |size| will be set to 0 and it means that the
```

Comment:
The comment `|size| will be set to 0` is inaccurate because of line 4013 in src/read.c:

```
4012             if (header.isSizeZeroBox) {
4013                 header.size = boxContents.size;
```

One way to make this comment accurate would be to set a local variable, say `boxSize`, to either `header.size` or `boxContents.size` and use `boxSize` instead of `header.size` in line 4025 in src/read.c. Another way would be to update the comment to also describe what we do in line 4013 in src/read.c.